assets/js/components/ExecutionResults/ExecutionResults.test.jsx

```diff
@@ -343,4 +344,82 @@ describe('ExecutionResults', () => {
     expect(screen.getAllByText('2/2 expectations passed')).toBeTruthy();
     expect(screen.getAllByText('1/2 expectations failed')).toBeTruthy();
   });
+
+  it('given provider is VMware, should render ExecutionResults with warning banner', async () => {
+    const {
+      clusterID,
+      hostnames,
+      loading,
+      catalog,
+      error,
+      executionLoading,
+      executionData,
+      executionError,
+      executionStarted,
+    } = prepareStateData('passing');
+
+    renderWithRouter(
+      <ExecutionResults
+        clusterID={clusterID}
+        clusterName="test-cluster"
+        clusterScenario="hana_scale_up"
+        cloudProvider="vmware"
+        hostnames={hostnames}
+        catalogLoading={loading}
+        catalog={catalog}
+        executionStarted={executionStarted}
+        catalogError={error}
+        executionLoading={executionLoading}
+        executionData={executionData}
+        executionError={executionError}
+      />,
+      { route: `/clusters/${clusterID}/executions/last?health=passing` }
```

Comment:
I don't know if this `route` does anything, but well, it is at least descriptive hehe